x-pack/plugins/security_solution/common/api/detection_engine/signals_migration/get_signals_migration_status/get_signals_migration_status.schema.yaml

```diff
@@ -0,0 +1,95 @@
+openapi: 3.0.0
+info:
+  title: Get alerts migration status API endpoint
+  version: '2023-10-31'
+paths:
+  /api/detection_engine/signals/migration_status:
+    post:
+      x-labels: [ess]
+      operationId: GetAlertsMigrationStatus
+      summary: Returns an alerts migration status
+      tags:
+        - Alerts migration API
+      parameters:
+        - name: from
```

Comment:
I don't see the codegen for this parameter?

Reply:
It's interesting when `x-codegen-enabled` is omitted then code generation works but doesn't generate all the entities. Since a valid spec may produce invalid code I explicitly added `x-codegen-enabled: true`.